tetris.py

```diff
@@ -192,5 +196,16 @@ def block_rotator(self, event):
             rotate.append((y, x))
         self.current_block = rotate
 
+    def full_line_clear(self):
+        y_coordinates = [y for (x, y) in self.landed_blocks]
+        coordinates_counter = collections.Counter(y_coordinates)
+        print(coordinates_counter)
+        for y_line, count in coordinates_counter.items():
+            if count == game_width:
+                self.landed_blocks = [(a, b) for (a, b) in self.landed_blocks if b != y_line]
+                time.sleep(0.5)
```

Comment:
Please don't use `time.sleep` in tkinter programs. It freezes the gui and does not improve anything. Delete this line.

Reply:
I tried playing the game with the `time.sleep`. It actually works quite fine. It's not the cleanest solution, but it works.